tests/puppeteer/pages/BO/international/languages/index.js

```diff
@@ -0,0 +1,119 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class Languages extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitle = 'Languages •';
+
+    // Header Selectors
+    this.localizationNavItemLink = '#subtab-AdminCurrencies';
```

Comment:
Here it's the same as `this.currenciesNavItemLink` shouldn't it be `#subtab-AdminParentLocalization` maybe if you target for the parent link

Reply:
fixed